src/odemis/acq/path.py

```diff
@@ -771,7 +771,7 @@ def getStreamDetector(self, path_stream):
             except AttributeError:
                 pass  # will raise error just after
 
-        raise LookupError("Failed to find a detector on stream %s" % (path_stream.name.value))
+        raise LookupError("Failed to find a detector on stream %s" % path_stream.name.value)
```

Comment:
No, add a comma. In general, this is an error pattern, where a tuple with a single element is written `(a)`, instead of the proper `(a,)`. Blame Python's weird syntax for that. Anyway, here, the point is to force a tuple, to ensure a safe behaviour of `%`.